app/controllers/api_controller.rb

```diff
@@ -12,4 +12,17 @@ def authenticated?
       end
     end
   end
+
+  def user_find
+    authenticate_or_request_with_http_basic do |email|
+      user = User.find_by_email(email)
+    end
+  end
+
+  def list_find
+    authenticate_or_request_with_http_basic do |email|
+      user = User.find_by_email(email)
+      list = user.lists.first
```

Comment:
hmm whenever I see a #first that makes me think it is a code smell. Do you want to loop through all of the lists?